_blog/articles/2024-01-13-cloud-service-categories.md

```diff
@@ -0,0 +1,202 @@
+            </li>
+            <li>You are responsible for:
+                <ul>
+                    <li>Configuration (of Application and Services)</li>
+                    <li>Application code/data (if needed)</li>
+                </ul>
+            </li>
+        </ul>
+    </li>
+</ul>
+
+</details>
+
+### SaaS (Software as a Service)
+
+![Image](/images/cloud-service-categories/SAAS-08-12-2023.gif "SaaS (Software as a Service)")
+
+<details>
+    <summary>Notes</summary>
+    <ul>
+    <li>Centrally hosted software (mostly on the cloud)
+        <ul>
+            <li>Offered on a subscription basis (pay-as-you-go)
+                <ul>
+                    <li>Examples:
+                        <ul>
+                            <li>Email, calendaring & office tools (such as Outlook 365, Microsoft Office 365, Gmail, Google Docs)</li>
+                            <li>Customer relationship management (CRM), enterprise resource planning (ERP), and document management tools</li>
+                        </ul>
+                    </li>
+                </ul>
+            </li>
+            <li>Cloud provider is responsible for:
+                <ul>
+                    <li>OS (incl. upgrades and patches)</li>
+                    <li>Application Runtime</li>
+                    <li>Auto scaling, Availability & Load balancing etc..</li>
+                    <li>Application code and/or Application Configuration (How much memory? How many instances? ..)</li>
+                </ul>
+            </li>
+            <li>Customer is responsible for:
+                <ul>
+                    <li>Configuring the software!</li>
+                    <li>Your data</li>
+                </ul>
+            </li>
+        </ul>
+    </li>
+</ul>
+
+</details>
+
+### Example 1: VMs with Load Balance
+
+![Image](/images/cloud-service-categories/Example1.gif "Example 1: VMs with Load Balance ")
+
+<details>
+    <summary>Notes</summary>
+    <ul>
+    <li>IaaS Approach
+        <ul>
+            <li>Setup VMs in Different Zones</li>
+            <li>Install Software on each VM</li>
+            <li>Install Application on each VM</li>
+            <li>Setup Load Balancer</li>
+        </ul>
+    </li>
+    <li>PaaS Approach
+        <ul>
+            <li>Use either AWS Elastic Beanstalk, Azure Web Apps, or Google App Engine to set this up!</li>
+        </ul>
+    </li>
+</ul>
+
+</details>
+
+### PAAS is NOT just for Compute Services
+
+![Image](/images/cloud-service-categories/PASS-isnt-just-compute.gif "PAAS is NOT just for Compute Services!")
+
+<details>
+    <summary>Notes</summary>
+    <ul>
+    <li>Compute: AWS Elastic Beanstalk, Azure App Service, Google App Engine</li>
+    <li>Databases: Relational & NoSQL (Amazon RDS, Google Cloud SQL, Azure SQL Database, etc.)</li>
+    <li>Queues</li>
+    <li>AI</li>
+    <li>ML</li>
+    <li>Operations</li>
+    <li>and a lot more...</li>
+</ul>
+
+</details>
+
+### Example 2: MySQL Database with a Standby
```

Comment:
Example 2: Deployment MySQL Database with High Availability

Do not change image (only heading in markdown)